app/form_models/publishers/job_listing/contract_information_form.rb

```diff
@@ -0,0 +1,33 @@
+class Publishers::JobListing::ContractInformationForm < Publishers::JobListing::VacancyForm
+  include ActiveModel::Attributes
+  attr_accessor :contract_type, :fixed_term_contract_duration, :working_patterns, :working_patterns_details
+
+  validates :contract_type, inclusion: { in: Vacancy.contract_types.keys }
+  validates :working_patterns, presence: true, inclusion: { in: Vacancy::WORKING_PATTERNS }
+  validates :fixed_term_contract_duration, presence: true, if: -> { contract_type == "fixed_term" }
+  validates :is_parental_leave_cover, inclusion: { in: [true, false] }, if: -> { contract_type == "fixed_term" }
+  validates :working_patterns, presence: true, inclusion: { in: Vacancy.working_patterns.keys - %w[job_share] }
```

Comment:
:raising_hand_man: The `:working_patterns` validation seems duplicated.

Reply:
Ah good spot, thanks